schemas/app/inventory/StockLedgerEntry.json

```diff
@@ -49,6 +49,13 @@
       "label": "Ref. Type",
       "fieldtype": "Data",
       "readOnly": true
+    },
+    {
+      "fieldname": "batchNumber",
+      "label": "Batch No",
+      "fieldtype": "Data",
```

Comment:
Why is this a **Data** type, shouldn't it be a **Link** type.

And why have you set create to `true`? `StockLedgerEntry` is not doc where manual entries are made.

Reply:
corrected